Tests/StencilSwiftKitTests/ContextTests.swift

```diff
@@ -13,38 +13,38 @@ class ContextTests: XCTestCase {
   func testEmpty() {
     let context = [String: Any]()
     
-    let result = try! enrich(context: context, parameters: [])
+    let result = try! StencilContext.enrich(context: context, parameters: [])
```

Comment:
Shouldn't we explicitly pass a `["PATH": "foo:bar:baz"]` value there for the `environment:` parameter, so that the unit test is deterministic and not dependent on the shell environment used by the test process? (even if `PATH` is really likely to be present in any shell environment, at least there it's explicit)?